docs/reference/monitoring/configuring-metricbeat.asciidoc

```diff
@@ -45,22 +45,22 @@ For more information, see <<monitoring-settings>> and <<cluster-update-settings>
 . {metricbeat-ref}/metricbeat-installation-configuration.html[Install {metricbeat}] on each
 {es} node in the production cluster.  Failure to install on each node may result in incomplete or missing results.
 
-. Enable the {es} {xpack} module in {metricbeat} on each {es} node.
+. Enable the {es} module in {metricbeat} on each {es} node.
 +
 --
-For example, to enable the default configuration in the `modules.d` directory, 
-run the following command:
+For example, to enable the default configuration for the {stack-monitor-features}
+in the `modules.d` directory, run the following command:
 
 ["source","sh",subs="attributes,callouts"]
 ----------------------------------------------------------------------
 metricbeat modules enable elasticsearch-xpack
 ----------------------------------------------------------------------
 
-Alternatively, you can use the {es} module, as described in the
-{metricbeat-ref}/metricbeat-module-elasticsearch.html[{es} module usage for {stack} monitoring]. 
+For more information, refer to 
+{metricbeat-ref}/metricbeat-module-elasticsearch.html[{es} module]. 
```

Comment:
Is it weird that we link to the other module here? That seems like a confusing thing for users too. We should either add docs for the `-xpack` modules (and maybe we do and I can't find them) or not link at all?

Reply:
Yes, it seems like the -xpack modules aren't documented.  I'll follow up on whether that's in the plans and remove the link if not.

Reply:
TBH I can't remember the history of this content, but I *think* the -xpack modules were documented originally but caused confusion. Now the docs have a section that mentions monitoring here: 

https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-module-elasticsearch.html#_usage_for_stack_monitoring_2

This suggests that you either enable `xpack.enabled: true` or enable the `elasticsearch-xpack` module. I think @ycombinator might know more.

Reply:
@chrisronline We re-added the "Usage for Stack Monitoring" section in the elasticsearch module docs (via https://github.com/elastic/beats/pull/23286) since it didn't seem possible to have separate (duplicate) docs for the elasticsearch-xpack module. If you still think this link is more confusing than helpful and ought to be removed from these instructions, just let me know!

Reply:
Sounds good to me!